consensus/parlia/parlia.go

```diff
@@ -399,40 +406,48 @@ func (p *Parlia) verifyVoteAttestation(chain consensus.ChainHeaderReader, header
 	if attestation == nil {
 		return nil
 	}
+	if attestation.Data == nil {
+		return fmt.Errorf("invalid attestation, vote data is nil")
```

Comment:
Better to have more useful information in the log here, like voteDate information

Reply:
vote data is nil, no more info?